app/assets/javascripts/student_profile/ProfileBarChart.js

```diff
@@ -70,24 +70,50 @@ export default class ProfileBarChart extends React.Component {
     });
   }
 
+  makePlotbands(monthKeys) {
```

Comment:
So this is reaching into a component called `ProfileBarChart` inside the `student_profile` folder for an experimental feature.  Ideally we'd keep these things separated, so that experimental code doesn't introduce more risk to core production features, and so that we don't have to maintain backwards compatibility with experimental code when refactoring or improving core production features.

To 🚢 this and keep moving, what do you think about copying this component file into the `service_types` folder with this patch, and keeping the existing one as-is.  That way it's fast to ship this feature, and we avoid coupling together these two separate parts of the app.